extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -23,4 +23,12 @@
      */
     @ConfigItem(defaultValue = "60")
     public Optional<Duration> defaultTransactionTimeout;
+
+    /**
+     * The directory name of location of the transaction logs.
+     * If the value is not absolute then the directory is relative
+     * to the <em>user.dir</em> system property.
+     * */
+    @ConfigItem(name = "objectStoreDir", defaultValue = "ObjectStore")
+    public String objectStoreDir;
```

Comment:
```suggestion
    public String objectStoreDirectory;
```
We favor consistency in Quarkus rather than consistency with the underlying technology.